src/west/commands/command.py

```diff
@@ -44,12 +44,20 @@ def __init__(self, **kwargs):
         super(ExtensionCommandError, self).__init__(**kwargs)
 
 
+_NO_TOPDIR_MSG_FMT = '''Error: "{}" is not in a west installation.
+Things to try:
+ - Set ZEPHYR_BASE to a zephyr repository path in a west installation.
+ - Run "west init" to set up an installation here.
+ - Run "west init -h" for additional information.
+'''
+
 class WestCommand(ABC):
     '''Abstract superclass for a west command.
 
     All top-level commands supported by west implement this interface.'''
 
-    def __init__(self, name, help, description, accepts_unknown_args=False):
+    def __init__(self, name, help, description, accepts_unknown_args=False,
+                 requires_installation=True):
```

Comment:
Are we breaking the `WestCommand` API here?

Reply:
It's a backwards compatible change.